src/__test__/connection/reconnect.test.ts

```diff
@@ -10,7 +10,7 @@ import {
 } from "@eventstore/db-client";
 
 // This test can take time.
-jest.setTimeout(120_000);
+jest.setTimeout(180_000);
```

Comment:
These tests were flaky before the change: https://github.com/EventStore/EventStore-Client-NodeJS/issues/246

I think the issue is that they are hanging forever, so increasing the timeout is futile. We'll need to try to write a more reliable test for this, but I don't think it needs to be done in this PR.

Reply:
Removed the timeouts